R/filetracker_roclet.R

```diff
@@ -0,0 +1,22 @@
+#' Create roclet that tracks Rust source files changes.
+#' @returns A roclet.
+#' @export
+filetracker_roclet <- function() {
+  roxygen2::roclet("filetracker")
+}
+
+#' @method roclet_process roclet_filetracker
```

Comment:
I believe `@method` is deprecated. Do you know whether it is needed here?

Reply:
I had problems with exporting this S3 method. The generic itself comes from roxygen2, and I need an `S3method` entry in the NAMESPACE of rextendr. Without it, generics are not resolved correctly.

Reply:
UPD: I found this topic here which deals with exactly the same problem:
https://github.com/r-lib/devtools/issues/1585
`@method` is not required as long as `document()` is executed twice. I believe this is the same problem that we have -- at the time of first compilation, there is no knowledge that `S3` method from `roxygen2` is imported and exported methods should be treated as such. During the second `document()` call, it is already known and the correct `S3` export is generated.

I do not know why it is still a thing, it looks like it has been fixed and the issue is several years old, yet here we are.